index.d.ts

```diff
@@ -118,14 +118,59 @@ declare module 'react-native-twilio-video-webrtc' {
     participant: Participant;
   };
 
-  export type MessageReceivedEventCb = ({ message: string, senderId: string }) => void;
-  export type StatsReceivedEventCb = (stats: any) => void;
+  export type MessageReceivedEventCb = ({
+    message: string,
+    senderId: string,
+  }) => void;
+  export type StatsReceivedEventCb = (stats: TwilioStats) => void;
 
   export type RoomEventCb = (p: RoomEventArgs) => void;
   export type RoomErrorEventCb = (t: RoomErrorEventArgs) => void;
 
   export type ParticipantEventCb = (p: ParticipantEventArgs) => void;
 
+  interface AudioTrackStats {
+    audioLevel: number;
+    jitter: number;
+  }
+
+  interface VideoTrackStats {
+    dimensions: {
+      width: number;
+      height: number;
+    }
+    frameRate: number;
+  }
+
+  interface BaseTrackStats {
+    codec: string,
+    packetsLost: number,
```

Comment:
We could potentially use `packetsLost` to show overlay with 'Poor connection' message to improve UX